### DISCUSSION TITLE
any new guidelines to add soapui plugin

### DISCUSSION THREAD

**Discussion by andavarv** [asdf-community/.github#58] [General]
I would like to add the below repo as part of this community.  

https://github.com/andavarv/asdf-soapui

Please let me know the process to follow

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)
